irohad/ametsuchi/impl/mutable_storage_impl.cpp

```diff
@@ -19,7 +19,7 @@
 #include "ametsuchi/impl/postgres_wsv_query.hpp"
 #include "ametsuchi/impl/redis_block_index.hpp"
 #include "model/commands/transfer_asset.hpp"
-
+#include "model/execution/command_executor_factory.hpp"  // for iroha::model::CommandExecutorFactory
```

Comment:
Do we need this comment?

Reply:
Comments help me to not forget why exactly this header is included. It is just a coincidence that header name and symbol name are equal. In many cases they are not.